README.md

```diff
@@ -99,26 +109,38 @@ $ yarn && yarn build && yarn start
 
   * To categorize the message under some tag you can use special tags in the format `:__some-tag:`.
 
-## 4 Server application
+## 5 Server application
 
-### 4.1 Set-up and running
+### 5.1 Set-up and running
 
-  * Create file `.env` in the `server` directory with the same content as you can find in `server/.env.template`. Then set values to variables `SLACK_APP_TOKEN` (see the section 3.1 above).
+  * In the `server` folder, copy `.env.template` into `.env` and fill missing values.
 
   * Go into the folder `server` and run the application:
 
 ```bash
-$ yarn && yarn build && yarn start
+$ yarn build && yarn start
 ```
 
   * Then you can run the Client application (see the following section).
 
-## 5 Client application
+## 6 Client application
 
-### 5.1 Set-up and running
+### 6.1 Set-up and running
 
-  * Got into the folder `client` and run the application:
+  * Go into the folder `client` and run the application:
 
 ```bash
-$ yarn && yarn build && yarn start
+$ yarn build
 ```
+
+## 7 Deployment
+
+We use [Heroku](https://www.heroku.com/home) for hosting this application. When something is merged into `production` branch, it is automatically deployed. You can also deploy any branch manually from Heroku Dashboard.
```

Comment:
Are you planning to add a `production` branch? or should this be `master` instead?

Reply:
I will add `production` branch once it will be in deployable state (after this PR is merged).

Reply:
What will be the difference between `master` and `production`? Isn't `master` our production?

Reply:
I agree, as we don't have stage env, I think master is enough

Reply:
Ok.